src/controllers/ownerMessageController.js

```diff
@@ -0,0 +1,64 @@
+const ownerMessageController = function (OwnerMessage) {
+  const postOwnerMessage = function (req, res) {
+    if (req.body.requestor.role !== 'Owner') {
+      res.status(403).send('You are not authorized to create messages!');
+    }
+    const ownerMessage = new OwnerMessage();
+    ownerMessage.message = req.body.newMessage;
+    ownerMessage.save().then(() => res.status(201).json({
+      _serverMessage: 'Message succesfuly created!',
+      ownerMessage,
+    })).catch(err => res.status(500).send({ err }));
+  };
```

Comment:
Works perfectly, still leaving a comment for the future if someone wants to optimize further. We can let go of the variable `const ownerMessage` and directly use an instance `OwnerMessage`

For example, here it could be written as:

```
 const ownerMessage = new OwnerMessage({ message: req.body.newMessage });
    try {
      await ownerMessage.save();
      res.status(201).json({
        _serverMessage: 'Message successfully created!',
        ownerMessage,
      });
    } catch (error) {
      res.status(500).send({ error });
    }
  };
```